src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/MessagingMultiNodeTestBase.kt

```diff
@@ -323,6 +351,23 @@ abstract class MessagingMultiNodeTestBase : IntegrationTestBase() {
         ),
     )
   }
+  private fun stubPrisonResponse(
```

Comment:
You now delete the 'PrisonEventListenerIntTest' stubPrisonReponse now and just use this